packages/node/src/handlers/call-api.ts

```diff
@@ -49,14 +43,15 @@ function buildOptions(
   apiCredentials: adapter.ApiCredentials[]
 ): adapter.BuildRequestOptions {
   // Include airnode metadata based on _relay_metadata version number
-  const parametersWithMetadata = addMetadataParameters(chain, aggregatedApiCall, reservedParameters);
+  const metadataParameters: adapter.Parameters = addMetadataParameters(chain, aggregatedApiCall, reservedParameters);
```

Comment:
Nit: I would change this function name to `buildMetadataParameters()` since it's not really doing any "adding"